src/main/java/uk/gov/ida/verifylocalmatchingserviceexample/configuration/DatabaseEngine.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.ida.verifylocalmatchingserviceexample.configuration;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import java.util.Arrays;
+
+public enum DatabaseEngine {
+    POSTGRESQL("postgresql", "classpath:db.migration.postgresql"),
+    SQLITE("sqlite", "classpath:db.migration.sqlite");
```

Comment:
Done this way rather than just putting the text into a string somewhere because if we do ever want to support a different database as well they may be able to use one of the existing migrations (various ones share the bit of syntax that differ between these two).